.github/workflows/scripts/init-rabbitmq.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+MAX_WAIT=60 # max. 1 minute waiting time in loop before timeout
```

Comment:
or `MAX_WAIT_SECONDS` and then it doesn't need a comment (I know it's just copied 😅 )

Reply:
And I think the original code was copied from faf stack lol.